examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -142,61 +191,71 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 					`Registered for webhook notifications at URL: "${subscriberUrl}".`,
 				),
 			);
-			result.send();
+		} catch (error) {
+			if (error instanceof ApiError) {
+				console.warn(formatLogMessage(error.message));
+				result.status(error.code).json({ message: error.message });
+			} else {
+				console.error(error);
+				throw error;
+			}
 		}
+
+		result.send();
 	});
 
-	interface UnregisterWebhookRequest extends Request {
-		body: {
-			// The target URL to unsubscribe from change notifications
-			url: string;
-		};
-	}
 	/**
-	 * Unregister's the sender's URL from receiving notifications when the external task-list data changes.
+	 * Unregisters the senders URL from receiving notifications when this endpoint is called at the end of a collaboration session
 	 *
-	 * Expected input data format: {@link UnregisterWebhookRequest}
+	 * Expected request body format: {@link UnregisterWebhookRequest}
```

Comment:
Same nit as above, I think the typing on the parameter is enough and we don't need this line

Reply:
Done, removed the line  in for both register & unregister in commit #a8d6dd5